### PR TITLE
feat: update playwright to test cross-browser (chromium, firefox, webkit)

### PR DESCRIPTION
**Pull request by Ibrahimmaga**
This feature will perform cross-browser testing
# Description
This feature will be testing multiple browser like Chromium, Firefox and Webkit.

1. Install playwright library
2. Create playwright-automation.js file
3. Create CI - Palaywright.yml

## Motivation & context
This feature will ensure that our application will support all three browsers 

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [x] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: .github/workflows/ci-playwright.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+name: CI - Playwright
```
Comment (awentzel) [inline .github/workflows/ci-playwright.yml:1]: Please rename to "CI - FAST Testing" to be consistent with other naming patterns used on GitHub Actions

File: .github/workflows/ci-playwright.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
```
Comment (awentzel) [inline .github/workflows/ci-playwright.yml:7]: Quotes are not needed and this then will be consistent with other branches used on GitHub Actions

File: .github/workflows/ci-playwright.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+jobs:
+  cross-browser-automation:
+    name: ${{ matrix.os }} Playwright@${{ matrix.playwright}} headless=${{ matrix.headless }}
+    strategy:
+      fail-fast: false
+
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-2019]
```
Comment (awentzel) [inline .github/workflows/ci-playwright.yml:16]: Is there a 'windows-latest', if so I suggest using it so we don't fall out of date.

File: .github/workflows/ci-playwright.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    - name: Prepare workspaces
+      run: yarn prepare
+
+    - name: Testing Prettier format
+      run: yarn format:check
+
+    - name: Testing unit tests
+      run: yarn lerna run test --stream
+
+      # Headless running is the same across OSes
```
Comment (awentzel) [inline .github/workflows/ci-playwright.yml:52]: I'm not seeing any value in this comment since the next line mentions headless

File: yarn.lock [hunk 5]
```diff
@@ -4241,7 +4241,7 @@
     terminal-link "^2.0.0"
     v8-to-istanbul "^4.1.3"
   optionalDependencies:
-    node-notifier "^9.0.0"
+    node-notifier "^6.0.0"
```
Comment (awentzel) [inline yarn.lock:4244]: Why are we rolling to an older version of node-notifier?  Is this necessary?

**Review by awentzel — CHANGES_REQUESTED** [review]
This is really cool to see.  Please make the requested changes to bring consistency with other Actions.  Otherwise, just had a few questions to understand more.

File: yarn.lock (hunk 5 above)
Comment (Ibrahimmaga) [reply yarn.lock:4244]: do you think we should keep the old version? it has a moderate severity. but keep the old version for now.

File: .github/workflows/ci-playwright.yml (hunk 1 above)
Comment (Ibrahimmaga) [reply .github/workflows/ci-playwright.yml:1]: cool

File: .github/workflows/ci-playwright.yml (hunk 2 above)
Comment (Ibrahimmaga) [reply .github/workflows/ci-playwright.yml:7]: oaky. should we run this on schedule too right? weekly maybe?

File: .github/workflows/ci-playwright.yml (hunk 4 above)
Comment (Ibrahimmaga) [reply .github/workflows/ci-playwright.yml:52]: true

**Comment by nicholasrice** [comment]
@Ibrahimmaga Have you verified this works with the playwright branch that @radium-v put together?

File: .github/workflows/ci-playwright.yml (hunk 3 above)
Comment (Ibrahimmaga) [reply .github/workflows/ci-playwright.yml:15]: There is window latest. but I put windows-2019 so we don't get windows 32 bit randomly. I believe window 2019 will give us 64 bit by default. I could be wrong.

Comment (awentzel) [reply .github/workflows/ci-playwright.yml:21]: If we get Win32 randomly I think that's ok.  There are plenty of potential users that very well could be using this on Win32.

File: yarn.lock (hunk 5 above)
Comment (awentzel) [reply yarn.lock:4244]: If this mitigates a security warning on a dependency then using 6 is great.  Thanks for the clarification.

**Comment by Ibrahimmaga** [comment]
This is for testing ground on playwright

**Pull request closed without merging**